README.md

```diff
@@ -26,5 +26,61 @@ I recommend using the open source [sqlitebrowser](http://sqlitebrowser.org/) to
 ![sqllitebrowser_screenshot](
 https://github.com/sqlitebrowser/sqlitebrowser/raw/master/images/sqlitebrowser.png)
 
+You can also upload the created `*.sql3` file to https://sqliteonline.com/, saving yourself from needing to download any applications.
+
+## Usage Examples
+
+### SQL
+
+- `SELECT * FROM  nutrient;` - print out all nutrient data
+- `SELECT Count(*) FROM  food;` - 8789 - all foods
+- `SELECT Count(*) FROM  food where food_group_id !=300 and food_group_id !=3500` - 8257 - Exclude all baby food and Native American food
+- `SELECT id FROM  food;` - [ [food.id] ]
+- `SELECT id,long_desc FROM  food;` - [ [food.id, food.long_desc] ]
+- `SELECT id,name FROM  nutrient;` - [ [nutrient.id, nutrient.name] ]
+- `SELECT count(*),food_group.name FROM food inner join food_group on food.food_group_id=food_group.id group by food_group_id; ` - the count of all the foods in each food group
+
+Every food's serving size, if it has one
+
+```sql
+    SELECT 
+      food.id,
+      food.long_desc,
+      gm_weight
+    FROM  food 
+    INNER JOIN weight ON weight.food_id=food.id
+    where weight.description=='serving';
+```
+
+Nutrient amounts per 100g of food
+
+```sql
+    SELECT 
+      food.id,
+      nutrient.name,
+      nutrition.amount
+    FROM  nutrition 
+    INNER JOIN food ON nutrition.food_id=food.id 
+    INNER JOIN nutrient ON nutrition.nutrient_id=nutrient.id;
+```
+
+### Post processing
+
+You can export this data to JSON and do further processing, like indexing. Indexing speeds up access for client Applications and reduces file size (by removing redundant data). Indexing the final SQL example reduces JSON representation from ~10MB to 7.0MB.
+
+```js
+    import Amt from './exportedUsdaJsonDataFromSqliteOnlineFinalExample';
```

Comment:
Whats contained in `exportedUsdaJsonDataFromSqliteOnlineFinalExample`? Each table in JSON?

Reply:
Nutrient amounts, it is the result of the final given query